### PR TITLE
Promotional offer ObjC API tester

### PR DESCRIPTION
**Pull request by taquitos**
Second part of [CF-563]

[CF-563]: https://revenuecats.atlassian.net/browse/CF-563?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Tests/APITesters/ObjCAPITester/ObjCAPITester/main.m [hunk 1]
```diff
@@ -7,15 +7,16 @@
 
 #import <Foundation/Foundation.h>
```
Comment (taquitos) [inline Tests/APITesters/ObjCAPITester/ObjCAPITester/main.m:8]: some sorting

File: Tests/APITesters/SwiftAPITester/SwiftAPITester/PromotionalOfferAPI.swift [hunk 2]
```diff
@@ -14,6 +14,6 @@ func checkPromotionalOfferAPI() {
     let discount: StoreProductDiscount = offer.discount
     let sk1Discount = offer.discount.sk1Discount
     let sk2Discount = offer.discount.sk2Discount
-
```
Comment (taquitos) [inline Tests/APITesters/SwiftAPITester/SwiftAPITester/PromotionalOfferAPI.swift:17]: Lint fix, also in base

**Review by NachoSoto — APPROVED** [review]

File: .circleci/config.yml [hunk 3]
```diff
@@ -182,6 +182,7 @@ jobs:
       - run:
           name: Run tests
           command: bundle exec fastlane test_ios
+          no_output_timeout: 30m
```
Comment (NachoSoto) [inline .circleci/config.yml:185]: I thought about this, cause that's the new thing that was making iOS 15 tests fail after I changed CI to use the new sims.
I guess as long as they're not taking forever this makes sense.

Comment (taquitos) [reply .circleci/config.yml:185]: @NachoSoto Hah, I just commented on another PR- CI seems to be restarting the sims and it's taking forever 🙃

**Pull request merged**